Spectral/style.css

```diff
@@ -0,0 +1,473 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    color: #fff;
+    font-family: "Open Sans", Helvetica, sans-serif;
+    font-weight: 400;
```

Comment:
why are you gave font-weight property to body it takes all the elements in the docu. don't use here